interface-definitions/interfaces_vxlan.xml.in

```diff
@@ -117,15 +117,35 @@
                 <format>u32:0-4094</format>
                 <description>Virtual Local Area Network (VLAN) ID</description>
               </valueHelp>
+              <valueHelp>
+                <format>&lt;start-end&gt;</format>
+                <description>VLAN ids range (use '-' as delimiter)</description>
```

Comment:
Let's keep the "ID" spelling consistently capitalized.

Reply:
Done